### ISSUE TITLE
Make Markor more accessible (accessibility)

### ISSUE THREAD

**Issue by schulle4u** [gsantner/markor#613]
#### General information

* **App version:** 2.0.1 F-Droid
* **System:** Android 8.1, Moto G4 Plus

#### Description

Currently the buttons for text editing and other controls are not accessible to users of assistive technology, e.g. Android Accessibility Suite (TalkBack). Even if announcement of button ID's is turned on in TalkBack settings, most buttons are still recognized as unlabeled, making them quite useless. This could be improvid by adding alternative texts for blind users.

**Comment by Andrejkaaa** [comment]
Hi. I'm interested to solve the issue #613. Could you please assign it to me? Thanks.

**Comment by gsantner** [comment]
I think this one needs some discussion. What actions to set and apply.

It's not something that can be fixed per se

**Comment by Andrejkaaa** [comment]
Can you please specific which buttons you mean?

**Comment by schulle4u** [comment]
@Andrejkaaa mainly the buttons in the editor window. It seems to be a kind of formating toolbar, but all buttons are completely unlabeled, and TalkBack users are not able to read out their functionality, even if reporting of element ID's is turned on in TalkBack settings. Some other buttons such as the one to create a new file at least has an element ID, but it is not very informative. TalkBack reports something like `UI_file_system_item_image`. 

Steffen

**Comment by gsantner** [comment]
Thanks @pvagner for your work! In next version Markor will be a lot better in terms of accessibility
